servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/OpenTelemetryHttpServerFilter.java

```diff
@@ -122,6 +122,6 @@ protected void tagStatusCode() {
      * @return the operation name to build the span with.
      */
     private static String getOperationName(HttpRequestMetaData metaData) {
-        return metaData.method().name() + ' ' + metaData.requestTarget();
+        return metaData.method().name() + ' ' + metaData.path();
```

Comment:
What changes in this case? Is this a breaking change?

Reply:
I highlight the reasons below but basically, this is too verbose and hard to filter:
<img width="525" alt="Screenshot 2023-03-10 at 9 54 21 AM" src="https://user-images.githubusercontent.com/115408/224388786-c0161ae5-9e67-4121-969f-5189f5e8fdb2.png">
Compared to this (Spring MVC):
<img width="513" alt="Screenshot 2023-03-10 at 9 54 08 AM" src="https://user-images.githubusercontent.com/115408/224388938-880c2ade-dbfd-4601-9fc8-33c1a31b3a2b.png">
Yes that last one had some query params but it still shows it in a more concise way. There's no specification on what it should look like but I think we should avoid having it too big to make it easy to filter or apply other things to it.